examples/custom_issue_manager.py

```diff
@@ -1,3 +1,5 @@
+from typing import Any, Dict, List, Optional
+
 import numpy as np
 import pandas as pd
```

Comment:
Need this import for my other suggestions:

```suggestion
import numpy as np
import numpy.typing as npt
import pandas as pd
```